tests/suite/test_suite.py

```diff
@@ -65,7 +65,9 @@
     e2e_logging.test,
     e2e_logging.test,
     e2e_logging.test,
+    e2e_logging.test,
     reconfiguration.test_add_node_from_snapshot,
+    e2e_logging.test_view_history,
```

Comment:
This actually tests that all nodes have the same view history. This test would have failed before this PR.